### ISSUE TITLE
distro: archlinux

### ISSUE THREAD

**Issue by bowling233** [ZJUSCT/jenkins.clusters.zjusct.io#4]

**Comment by YooLc** [comment]
we need more distros used in real competitions :smiling_imp: 
eg. OpenEuler, Rocky Linux, ....

**Comment by YooLc** [comment]
提交在这里: https://github.com/ZJUSCT/jenkins.clusters.zjusct.io/commit/81c3d648ee45b8be5fb380cb603169edd184764a 👿

https://github.com/ZJUSCT/jenkins.clusters.zjusct.io/blob/81c3d648ee45b8be5fb380cb603169edd184764a/archlinux/chroot-installation.sh#L2-L4

**Comment by bowling233** [comment]
d8e1f04100dcd600c6621537f93dd46e520bde43 fix [Jenkins Build #2](https://jenkins.clusters.zjusct.io/job/chroot-archlinux/2/)

```text
+ pacman --noconfirm -S archlinuxcn-keyring vim nano git gdb gcc clang cmake htop iperf3 net-tools cpufetch doxygen exfatprogs fish git-extras git-lfs hwinfo hwloc ipmitool iptraf-ng jq lldb lldpd mc meson pipewire-jack fastfetch networkmanager ninja numactl nvtop proxychains-ng screenfetch sudo tcpdump tldr traceroute tree termshark unrar valgrind vim wget wireshark-cli zip squashfs-tools p7zip hugo go rust python-pipx qemu-full bridge-utils zfs-utils aria2 bat btop duf fq fzf fish gdu httpie hyperfine ripgrep tmux zoxide zsh starship aarch64-linux-gnu-gcc riscv64-linux-gnu-gcc autoconf automake curl bison flex texinfo gperf libtool patchutils bc gdb-multiarch tcpdumpecho xxxxxSUCCESSxxxxx
warning: curl-8.11.0-3 is up to date -- reinstalling
error: target not found: tcpdumpecho
error: target not found: xxxxxSUCCESSxxxxx
```

**Comment by bowling233** [comment]
87f553849d90d9e14d8800783a36376049e18baa now works perfectly! 🎉🎉🎉
